runner/main.js

```diff
@@ -16,6 +16,14 @@ function createTemporaryDirectory() {
   return directoryPath;
 }
 
+function extendedErrorMessage(request, error) {
+  const stackTrace = new Error().stack;
+  // Chop off first two lines which are "Error:" and this method.
+  // Also, anything from XMLHttpRequest.send and above is not useful
+  const revisedTrace = stackTrace.replace(/\s+at XMLHttpRequest.send[\s\S]*/m, "").split("\n").slice(2).join("\n");
+  return "ERROR: " + error.message + "\r\n" + revisedTrace + "\r\n  Failed request: " + JSON.stringify(request) + "\r\n" ;
```

Comment:
Is `\r\n` required here or would `\n` work as well?

Reply:
I was developing on MacOS, where "/n" works fine. I imagine that on Windows, Deno is probably going to do the right thing with linebreaks in strings, but I haven't tried it on that platform.